db/arango/csvGenerator.js

```diff
@@ -0,0 +1,132 @@
+let faker = require('faker');
```

Comment:
There's a lot of repeated code here. Just as a suggestion: When you have time, you should refactor it and use a switch/case to deal with different object types.

Reply:
I found 2-3 lines that can be put together in a function, will refactor it later. Thank!